ml-dsa/Cargo.toml

```diff
@@ -15,14 +15,17 @@ categories = ["cryptography"]
 keywords = ["crypto", "signature"]
 
 [features]
-default = ["std", "rand_core"]
-std = ["sha3/std"]
+default = ["std", "rand_core", "alloc"]
+std = ["alloc", "sha3/std"]
 zeroize = ["dep:zeroize", "hybrid-array/zeroize"]
 rand_core = ["dep:rand_core", "signature/rand_core"]
+alloc = ["pkcs8/alloc"]
 
 [dependencies]
+const-oid = { version = "0.10.0-rc.1", features = ["db"] }
 hybrid-array = { version = "0.2.3", features = ["extra-sizes"] }
 num-traits = "0.2.19"
+pkcs8 = { version = "=0.11.0-rc.1", default-features = false }
```

Comment:
Elsewhere we don't make these hard dependencies. Perhaps make them optional with a `pkcs8 = ["dep:const-oid", "dep:pkcs8"]` feature? They can be on-by-`default`.